server/chat-commands/moderation.ts

```diff
@@ -1000,11 +1001,12 @@ export const commands: ChatCommands = {
 			userReason = target.substr(0, proofIndex).trim();
 		}
 
-		targetUser.popup(`|modal|${user.name} has globally banned you.${(userReason ? `\n\nReason: ${userReason}` : ``)} ${(Config.appealurl ? `\n\nIf you feel that your ban was unjustified, you can appeal:\n${Config.appealurl}` : ``)}\n\nYour ban will expire in a few days.`);
+		const {punishment, affected} = await Punishments.ban(targetUser, null, null, false, userReason);
+
+		targetUser.popup(`|modal|${user.name} has globally banned you.${(userReason ? `\n\nReason: ${userReason}` : ``)} ${(Config.appealurl ? `\n\nIf you feel that your ban was unjustified, you can appeal:\n${Config.appealurl}` : ``)}\n\nYour ban will expire ${Punishments.getPunishmentExpirationDescription(punishment)}.`);
```

Comment:
```suggestion
		targetUser.popup(
			`|modal|${user.name} has globally banned you.` +
			(userReason ? `\n\nReason: ${userReason}` : ``) +
			(Config.appealurl ? `\n\nIf you feel that your ban was unjustified, you can appeal:\n${Config.appealurl}` : ``) +
			`\n\nYour ban will expire ${Punishments.getPunishmentExpirationDescription(punishment)}.`
		);
```

Just bikeshedding, but we might as well make this more readable.